lib/src/paint/light_paint.dart

```diff
@@ -50,15 +51,16 @@ class LightPaint extends CustomPainter {
       ..close();
 
     canvas.drawPath(
-      circleHole,
+      circleHole!,
       Paint()
         ..style = PaintingStyle.fill
         ..color = colorShadow.withOpacity(opacityShadow),
     );
   }
 
   @override
-  bool shouldRepaint(LightPaint oldDelegate) {
-    return oldDelegate.progress != progress;
-  }
+  bool hitTest(Offset position) => circleHole!.contains(position);
+
+  @override
+  bool shouldRepaint(LightPaint oldDelegate) => true;
```

Comment:
This is marked as true always for two reasons:

- When the target had a `pulseAnimation` different from `1 to 1`, as for example the default `1 to 0.99`, this is always true, so in that case, it will not impact in any way.

- When the target had a `pulseAnimation` with a value of `1 to 1`, i.e. no pulse animation, the path object is sometimes null, so the hitTest cannot be recognized correctly, and it also caused that for this type of target, if it was moved or resized the hole always stayed in the same place with the same size. This avoids that